### DISCUSSION TITLE
Deploy quarto on node.js server

### DISCUSSION THREAD

**Discussion by TJ-HOU** [quarto-dev/quarto-cli#998] [Q&A] [answered]
Hi All, I am developing a website. I wrote the scripts in R Studio and saved as qmd files. How can I deploy the scripts on a server supported by node.js?

**Comment by jmbuhr** [comment, marked as answer]
First you will want to call `quarto render` and then take the resulting folder and serve it with your server. Here is the documentation for it: https://quarto.org/docs/websites/publishing-websites.html

**Reply by jmbuhr** [reply]
Any server than can serve static html files will do

**Reply by TJ-HOU** [reply]
Got it. Thanks a lot!
